package.json

```diff
@@ -17,7 +17,13 @@
   },
   "homepage": "https://github.com/GFDRR/thinkhazard",
   "dependencies": {
+    "bootlint": "^0.11.0",
```

Comment:
We should stick to one strategy: either set the versions or use the latest available.